packages/e2e-tests/specs/editor/blocks/quote.test.js

```diff
@@ -98,14 +100,15 @@ describe( 'Quote', () => {
 			await insertBlock( 'Quote' );
 			await page.keyboard.press( 'ArrowRight' );
 			await page.keyboard.type( 'cite' );
-			await transformBlockTo( 'Paragraph' );
+			await transformBlockTo( 'Unwrap' );
```

Comment:
This test is failing because after `ArrowRight` you can't type a `cite`. See https://github.com/WordPress/gutenberg/pull/25892. We need to either fix this keyboard interaction or adapt the test.